packages/strapi-plugin-users-permissions/config/queries/bookshelf.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
   },
 
   findOne: async function (params, populate) {
-    const primaryKey = params[this.primaryKey] || params.id;
+    const primaryKey = params[this.primaryKey] || params._id;
```

Comment:
Can we create small utility function that returns the correct primary key, because I saw this logic repeated several times:
```JS
function getPrimaryKey(params, defaultKey) {
   return params[defaultKey] || params.id || params._id;
}
```
This way we'll have a consistent way to retrieve the id.

Reply:
👍